pkgs/applications/networking/browsers/firefox/wrapper.nix

```diff
@@ -64,7 +65,7 @@ let
           ++ lib.optional (cfg.enableFXCastBridge or false) fx_cast_bridge
           ++ extraNativeMessagingHosts
         );
-      libs =   lib.optional stdenv.isLinux udev
+      libs =   lib.optionals stdenv.isLinux [ udev libva ]
```

Comment:
Is this stil required given that we are already building firefox with that lib?

Reply:
On `master` I don't see a direct dependency on `libva`.
(@colemickens mentioned that this is currently for the binary package only and I don't think our source build supports VA-API yet)

Reply:
This PR adds that support: https://github.com/NixOS/nixpkgs/pull/81917/files#diff-b39b9a678f9ecf5d0e1581832f7bf99dR26

Reply:
Oh, sorry. My bad, I didn't catch up with the third commit (also in the first change).

Reply:
If I drop it from the wrapper it seems like it can't find it at runtime?
`   1 [Child 20523: MediaPDecoder #2]: D/PlatformDecoderModule libva library is missing`